src/main/drivers/motor.c

```diff
@@ -133,8 +133,9 @@ static void analogInitEndpoints(const motorConfig_t *motorConfig, float outputLi
         *deadbandMotor3dLow = flight3DConfig()->deadband3d_low;
     } else {
         *disarm = motorConfig->mincommand;
-        *outputLow = motorConfig->minthrottle;
-        *outputHigh = motorConfig->maxthrottle - ((motorConfig->maxthrottle - motorConfig->minthrottle) * (1 - outputLimit));
+        const float minThrottle = motorConfig->mincommand + motorConfig->motorIdle * 0.1f;
```

Comment:
`mincommand` can go from 750 to 2250. If this change expects `mincommand` to go from 1000 to 2000, this breaks the old behavior.

Reply:
Previously, `minthrottle` could be configured anywhere from `PWM_PULSE_MIN` to `PWM_PULSE_MAX`, i.e. 750 to 2250.
Settings anywhere near the upper end are pointless, of course :-)
Previously, and without change in this PR, `mincommand` can also be set to the same range.  This PR *does not* expect mincommand to be any particular value (other than some reasonable value as configured by the user, or the limited range for brushless). 
After adding the idle amount, it will not be possible for our effective minThrottle to go down to 750, that's true, but for idle to work, if `mincommand` can't go below 750, then the idle value must be above that.
I think the code is correct.

Reply:
A comment implies that some analog ESC won't work unless you can go down to under 900, so I'm not so sure this works out